src/vector-crypto.adoc

```diff
@@ -269,6 +269,17 @@ Element Group Size (`EGS`).
 LMUL constraints::
 For element-group instructions, `LMUL`*`VLEN` must always be at least as large as `EGW`, otherwise an
 _illegal instruction exception_ is raised, even if `vl`=0.
++
+A *Vector Element Group* operand or destination sets `EMUL = LMUL`.
++
+A *Scalar Element Group* operand or destination sets `EMUL = ceil(EGW / VLEN)`.
++
+The standard RISC-V Vector mandates on vector register group indices alignment to `EMUL` apply: operand or destination register indices must be aligned to `EMUL`, all other non-aligned cases are __reserved__.
```

Comment:
I don't think this sentence is necessary, since the rule here is the same as for the V extension.  IMO, the redundancy reduces clarity, since readers might wonder if this is a special case, when of course it is not.

Reply:
```suggestion
```